packages/voorbeeld-design-tokens/documentation/number-badge/utrecht.numberBadge.stories.tsx

```diff
@@ -0,0 +1,27 @@
+import { Meta, StoryObj } from '@storybook/react';
+import { BadgeCounter } from '@utrecht/component-library-react/dist/css-module';
+
+const meta = {
+  id: 'utrecht-badge-counter-group',
```

Comment:
Dit moet 'utrecht-number-badge' zijn, en dat + `.stories.tsx` is ook de naam van deze file.